src/drug_detection_crawler/storage/save_tools.py

```
@@ -115,6 +115,15 @@ def append_csv_row(csv_path: str, row: dict) -> None:
 
         writer.writerow(row)
 
+def get_last_num(rows: list[dict]) -> int:
+    nums = []
+    for row in rows:
+        try:
+            nums.append(int(row.get("num", 0)))
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)